recipes-bsp/bootfiles/bootfiles.bbappend

```diff
@@ -1,5 +1,6 @@
-RPIFW_DATE = "20200122"
-SRCREV = "01ecfd2ba2b7cf3a2f4aa75ada895ee4a3e729f5"
-RPIFW_S = "${WORKDIR}/git"
-RPIFW_SRC_URI = "git://github.com/raspberrypi/firmware.git;protocol=git;branch=master"
+RPIFW_DATE = "20210421"
```

Comment:
Is this file dependent on the kernel version?

Reply:
Added this file under BBMASK to select based on kernel version

Reply:
The change with BBMASK is ok, but shouldn't we include the old version of the 'recipes-bsp/bootfiles/bootfiles.bbappend', so other kernel version does not break? Or wasn't this file necessary at all previously?

Reply:
In dunfell it will not work, so added additional bootfiles bbappend to get work in dunfell case